reviewers/GrammarChecker.js

```diff
@@ -77,14 +92,15 @@ checker.prototype = {
       
       for ( i = 0; i < quotedStrings.length; i++ ) {
         findQuotedStrings = quotedStrings[i];
-      
-        while ( result = findQuotedStrings.exec( line ) ) {
+        
+        result = findQuotedStrings.exec( line );
+        while ( result ) {
           nothingToDo = false;
           
```

Comment:
Don't make functions within a loop
```javascript
          (function GrammarCheckerStepClosure( composedLine ) {
```
'composedLine' is already declared in the upper scope.
```javascript
          (function GrammarCheckerStepClosure( composedLine ) {
```